src/main/java/appeng/me/storage/CellInventory.java

```diff
@@ -444,16 +446,62 @@ protected void loadCellItems()
 
 		for( int x = 0; x < types; x++ )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS135)